src/sql/workbench/contrib/resourceViewer/browser/resourceViewerTable.ts

```diff
@@ -93,4 +104,28 @@ export class ResourceViewerTable extends Disposable {
 		}
 		return value;
 	}
+
+	private async onTableClick(event: ITableMouseEvent): Promise<void> {
+		const column = this._resourceViewerTable.columns[event.cell.cell];
+		if (column) {
+			const row = this._dataView.getItem(event.cell.row);
+			const value = row[column.field];
+			if (isHyperlinkCellValue(value)) {
+				if (isString(value.linkOrCommand)) {
+					try {
+						await this._openerService.open(value.linkOrCommand);
+					} catch (err) {
+						vscode.window.showErrorMessage(localize('resourceViewerTable.openError', "Error opening link : {0}", err));
```

Comment:
@alanrenmsft - Note to self - be better about actually testing stuff before submitting, vscode API isn't available in core so this was just going to break 🤦‍♂️